DEVELOPMENT.md

```diff
@@ -80,3 +80,31 @@ Starting application at 127.0.0.1:3000
 
 And you'll be up and running. As you edit and save code, the `buffalo dev` command will notice and automatically
 re-compile and restart the server.
+
+# Run unit tests
+
+In order to run unit tests, services they depend on must be running first:
+
+```console
+make alldeps
+```
+
+and env vars exported:
+```console
+export GO_ENV=test_postgres
+export POP_PATH=$PWD/cmd/proxy
+export ATHENS_MONGO_STORAGE_URL=mongodb://127.0.0.1:27017
+export TMPDIR=/tmp/
```

Comment:
This all assumes linux, and a root level directory `/tmp`

Reply:
Perhaps we could create a project level `.env` file with all variables required for tests, then have the test scripts source that. As well, there is an issue regarding the `TMPDIR` variable. #447

Reply:
as this is doc file it's ok

Reply:
but yes., i would as well like to see script file which presets all required env variables

Reply:
6c4edbc do you think it is fine, or should I create .env in the project root folder ?